precode.sh

```diff
@@ -6,38 +6,55 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir task task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
При использовании флага `-p` можно сразу так делать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```